SIAB/spillage/jy_expmt.py

```
@@ -0,0 +1,203 @@
+        # w = wk[isk % len(wk)]
+
+        wfc, _, _, _ = read_wfc_lcao_txt(os.path.join(outdir, f"{fwfc}{isk+1}.txt"))
+        # the complete return list is (wfc.T, e, occ, k)
+        ovlp = read_triu(os.path.join(outdir, f"data-{isk}-S"))
+
+        # for monomer, it is okay to use wll method to decompose its components
+        wll = _wll(wfc, ovlp, running["natom"], running["nzeta"])
+
+        for ib, wb in enumerate(wll): # loop over bands
+            wlb = np.sum(wb.real, 1) # sum over one dimension, get dim 1 x lmax matrix
+            for l, wl in enumerate(wlb):
```

Ruff

Ambiguous variable name: `l`

(E741)